recipes-kernel/linux/files/patches-5.10/0044-arm64-dts-ti-iot2050-Flip-mmc-device-ordering-on-Adv.patch

```diff
@@ -9,6 +9,9 @@ devices, also avoiding surprises for users coming from the downstream
 kernels.
 
 Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
+Acked-by: Aswath Govindraju <a-govindraju@ti.com>
+Signed-off-by: Nishanth Menon <nm@ti.com>
+Link: https://lore.kernel.org/r/fe20d6346f119a28e47d129b616682001299cf0e.1632657917.git.jan.kiszka@web.de
```

Comment:
What was the update procedure? Just to explain the patch metadata diffs.

Reply:
I have completed the driver update on stable/v01.03
However, when I went to the master, I found that several patches had already used the upstream ones. I picked these patches again. It seems that I missed this one.
I will update this patch.

Reply:
Hi jan
I checked my patches again.  The signature on patch 44 was not displayed by github.